set.mm

```diff
@@ -638670,6 +638670,162 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       WHAVKVLVNCAGDVEZABWEWFVBWGVFVGVHAVSWNABVRWEWFAVKVLCWOWGVIVJVGTTT $.
   $}
 
+  ${
+    $d A x $.  $d A y $.  $d B x $.  $d B y $.  $d ph x $.  $d ph y $.
+    dvrelog2.1 $e |- ( ph -> A e. RR ) $.
+    dvrelog2.2 $e |- ( ph -> B e. RR ) $.
+    dvrelog2.3 $e |- ( ph -> 0 < A ) $.
+    dvrelog2.4 $e |- ( ph -> A <_ B ) $.
+    dvrelog2.5 $e |- F = ( x e. ( A [,] B ) |-> ( log ` x ) ) $.
+    dvrelog2.6 $e |- G = ( x e. ( A (,) B ) |-> ( 1 / x ) ) $.
```

Comment:
In this and the following theorems:
* define $F$ on $(A, B)$;
* $0 < A$ can be weakened to $0 \leq A$;
* $A \leq B$ is not needed;
* $A \in \mathbb{R}$ and $B \in \mathbb{R}$ could also be removed, but since that fact relies on our encoding details, it is better to keep them.

Reply:
Actually that is dvrelog3. I need to apply the fundamental theorem of calculus on it and in that case A will be 4 or something like that. Since those are just auxiliary lemmas that I need for that specific case and I wasn't aware of any conditions that will be of future use I implemented two lemmas. It's likely that an unused lemma will be deleted once the dependent theorem is formalised.

Reply:
I'm also not sure what you mean by defining F on (A,B) I assume that means the open interval. In that case dvrelog3 is the theorem that implements that one. In any case it is very likely that one of those theorems will be removed.